### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by AndreHaueisen**
This update is necessary because [ffmpeg_kit_flutter_full](https://pub.dev/packages/ffmpeg_kit_flutter) `6.0.3` has an updated gradle version, which we need to complete our dependency update.

### CODE REVIEW

File: lib/domain/bloc/controller.dart
Comment (AndreHaueisen) [inline lib/domain/bloc/controller.dart:494]: Normalize path before extracting the extension which should solve [#79](https://github.com/dart-lang/path/issues/79)

**Review by ebjorklund01 — APPROVED** [review]
Sorry this PR slipped through for a while. I pushed up a couple commits to this branch. 

I was running into the error 
`.pub-cache/hosted/pub.dev/helpers-1.2.0/lib/helpers/misc.dart:127:7: Error: No named parameter with the name 'window'.`

When running the example app. Give the example app a spin with these changes and go ahead and merge this once you confirm that it looks good on your end.

**Comment by AndreHaueisen** [comment]
No worries, I'm gonna merge it after we merge the dependency updates branch

**Pull request merged**
